src/olympia/addons/templates/addons/impala/details.html

```diff
@@ -145,6 +146,25 @@ <h4 class="author">{{ _('by') }} {{ users_list(addon.listed_authors) }}</h4>
         {% endwith %}
       </div>
     {% endif %}
+  {% else %}
+    {% if addon.contributions %}
+      <div class="notice c author">
+        <div class="button-wrapper">
+          <a class="button contribute prominent" id="contribution-url-button"
+             href="{{ addon.contributions|external_url }}">
```

Comment:
Since this is going to take you to a different page instead of opening a popin, maybe we want this in a new tab? If so, it'd need `target="_blank" rel="noopener"`. We don't do that often these days though, so I'm not sure.

Reply:
yeah, should be - the issue specified a new tab.

Reply:
https://github.com/mozilla/addons-server/pull/6274/commits/3896ee85e68cee6a2fbe43644a634f61a99687f6